src/windows/View.tsx

```diff
@@ -0,0 +1,371 @@
+        }
+
+        if (props.onMouseOver) {
+            if (this._onMouseOver === undefined) {
+                this._onMouseOver =  (e: React.SyntheticEvent<any>) => {
+                    if (this.props.onMouseOver) {
+                         this.props.onMouseOver(EventHelpers.toMouseEvent(e));
+                    }
+                };
+            }
+            this._internalProps.onMouseOver = this._onMouseOver;
+        }
+
+        if (props.onMouseMove) {
+            if (this._onMouseMove === undefined) {
+                this._onMouseMove =  (e: React.SyntheticEvent<any>) => {
+                    if (this.props.onMouseMove) {
+                         this.props.onMouseMove(EventHelpers.toMouseEvent(e));
+                    }
+                };
+            }
+            this._internalProps.onMouseMove = this._onMouseMove;
+        }
+    }
+
+    render(): JSX.Element {
+
+        let content: JSX.Element = super.render();
+
+        // Exit fast if the keyboard enabled component is not available
+        if (!HasFocusableWindows) {
+            return content;
+        }
+
+        if (this.props.tabIndex !== undefined) {
+            let tabIndex: number = this.getTabIndex() || 0;
+            let windowsTabFocusable: boolean =  tabIndex >= 0;
+
+            // RNW.FocusableWindows doesn't participate in layouting, it basically mimics the position/size of the child
+            let focusableViewProps: RNW.FocusableProps = {
+                ref: this._onFocusableRef,
+                isTabStop: windowsTabFocusable,
+                tabIndex: tabIndex,
+                disableSystemFocusVisuals: false,
+                handledKeyDownKeys: DOWN_KEYCODES,
+                handledKeyUpKeys: UP_KEYCODES,
+                onKeyDown: this._onFocusableKeyDown,
+                onKeyUp: this._onFocusableKeyUp,
+                onFocus: this._onFocus,
+                onBlur: this._onBlur
+            };
+
+            content = (
+                <RNW.FocusableWindows
+                    {...focusableViewProps}
+                >
+                    {content}
+                </RNW.FocusableWindows>
+            );
+        }
+
+        return content;
+    }
+
+    private _onFocusableRef = (btn: RNW.FocusableWindows): void => {
+        this._focusableElement = btn;
+    }
+
+    focus() {
+        super.focus();
+        // Only forward to Button.
+        // The other cases are RN.View based elements with no meaningful focus support
+        if (this._focusableElement) {
+            this._focusableElement.focus();
+        }
+    }
+
+    blur() {
+        super.blur();
+        // Only forward to Button.
+        // The other cases are RN.View based elements with no meaningful focus support
+        if (this._focusableElement) {
+            this._focusableElement.blur();
+        }
+    }
+
+    getChildContext() {
+        // Let descendant Types components know that their nearest Types ancestor is not an Types.Text.
```

Comment:
>Types [](start = 26, length = 5)

`Types` should be `RX`

Reply:
ok

---
In reply to: [159359502](https://github.com/Microsoft/reactxp/pull/404#discussion_r159359502) [](ancestors = 159359502)